charts/datadog/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: datadog
-version: 3.30.6
+version: 3.30.7
```

Comment:
I think we need to update the minor version when a parameter is added

Reply:
Fixed in [e9615a0](https://github.com/DataDog/helm-charts/pull/1058/commits/e9615a0afc93846fa187050ec5c9ff4e837885e5)